cmd/gotk/reconcile_kustomization.go

```diff
@@ -94,10 +94,10 @@ func reconcileKsCmdRun(cmd *cobra.Command, args []string) error {
 		logger.Actionf("annotating kustomization %s in %s namespace", name, namespace)
 		if kustomization.Annotations == nil {
 			kustomization.Annotations = map[string]string{
-				consts.ReconcileAtAnnotation: time.Now().Format(time.RFC3339Nano),
+				meta.ReconcileAtAnnotation: time.Now().Format(time.RFC3339Nano),
 			}
 		} else {
-			kustomization.Annotations[consts.ReconcileAtAnnotation] = time.Now().Format(time.RFC3339Nano)
+			kustomization.Annotations[meta.ReconcileAtAnnotation] = time.Now().Format(time.RFC3339Nano)
```

Comment:
Same for the logic below, although the current is less problematic given how the kustomize-controller handles state. It could however take the new `LastHandledReconcilateAt` status field into account, and also make use of `meta.GetCondition()`.